src/Interfaces/Options/Shape/IOptionsShapePolygon.ts

```diff
@@ -0,0 +1,3 @@
+export interface IOptionsShapePolygon {
+    nb_sides: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'nb_sides' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)